README.md

```diff
@@ -111,12 +112,31 @@ location @extensionless-php {
 }
 ```
 
+If you've put your nginx configuration in a file in `sites_available`, link it and test nginx
+
+```sh
+function link_nginx(){
+  domain="$1"
+  sudo ln -s /etc/nginx/sites-available/$domain /etc/nginx/sites-enabled/$domain
+}
+link_nginx your_domain.com
+sudo nginx -t
+```
+
 Start php-fpm and nginx immediately and on every boot:
 
 ```sh
 sudo systemctl enable --now php-fpm nginx
 ```
 
+The name of the php-fpm file might vary depending on the version; you can find it with
+
+```
+sudo systemctl list-unit-files | grep php.*fpm
+# example output: php8.1-fpm.service
+sudo systemctl enable --now php8.1-fpm
```

Comment:
probably worth removing the sudo from the line to keep in line with the assuming that everything is being configured as root. Maybe rather than checking the version, we should instruct the user to install a specific version, perhaps the latest supported?